internal/seclang/directives.go

```diff
@@ -636,14 +659,21 @@ func directiveSecAuditLogParts(options *DirectiveOptions) error {
 // Syntax: SecAuditEngine RelevantOnly
 // Default: Off
 // ---
-// The SecAuditEngine directive is used to configure the audit engine, which logs complete transactions. ModSecurity is currently able to log most, but not all transactions. Transactions involving errors (e.g., 400 and 404 transactions) use a different execution path, which ModSecurity does not support.
+// The SecAuditEngine directive is used to configure the audit engine, which logs complete
+// transactions. ModSecurity is currently able to log most, but not all transactions.
+// Transactions involving errors (e.g., 400 and 404 transactions) use a different execution
+// path, which ModSecurity does not support.
```

Comment:
```suggestion
// transactions.
```
nit: I think it is ModSecurity, or Apache/connector specific, I would remove it